blocks/blockoracle.go

```diff
@@ -65,14 +67,11 @@ func (bo *Oracle) BlockEligible(layerID types.LayerID) (types.ATXID, []types.Blo
 	if !bo.isSynced() {
 		return types.ATXID{}, nil, nil, fmt.Errorf("cannot calc eligibility, not synced yet")
 	}
+
 	epochNumber := layerID.GetEpoch()
 	bo.log.Info("asked for eligibility for epoch %d (cached: %d)", epochNumber, bo.proofsEpoch)
-	if epochNumber.IsGenesis() {
-		bo.log.Warning("asked for eligibility for epoch 0, cannot create blocks here")
-		return *types.EmptyATXID, nil, nil, nil
-	}
```

Comment:
Why remove this check?

Reply:
As eligibility proofs cannot be calculated in the genesis epochs, this check seemed redundant to me. However, having it seems more consistent now, so I'm going to revert this.